poetry/console/commands/env/remove.py

```diff
@@ -1,21 +1,36 @@
 from cleo import argument
+from cleo import option
 
 from ..command import Command
 
 
 class EnvRemoveCommand(Command):
 
     name = "remove"
-    description = "Removes a specific virtualenv associated with the project."
+    description = "Removes specific virtualenvs associated with the project."
```

Comment:
```suggestion
    description = "Removes specific virtual environments associated with the project."
```

While we are at it might as well improve the wording.